source/cl/test/UnitCL/source/Common.cpp

```diff
@@ -778,7 +778,7 @@ bool UCL::isImageFormatSupported(cl_context context,
       CASE(CL_RGx);
       CASE(CL_RGBx);
       CASE(CL_DEPTH);
-      CASE(CL_DEPTH_STENCIL);
+      // CASE(CL_DEPTH_STENCIL);
```

Comment:
If we want to leave it commented out can we have a comment (or just delete it). Same below.

Reply:
This is consistent with how it is done in modules/compiler/builtins/libimg/source/host.cpp. But I don't mind deleting it entirely if we do it in both places.

Reply:
I'm happy with deleting in both.

Reply:
Alright, done.